src/crewai/cli/cli.py

```diff
@@ -26,8 +26,13 @@
 
 
 @click.group()
+@click.version_option(pkg_resources.get_distribution("crewai").version)
 def crewai():
-    """Top-level command group for crewai."""
+    """Top-level command group for crewai.
+    
+    Options:
```

Comment:
![Screenshot 2024-12-02 at 10 34 48 AM](https://github.com/user-attachments/assets/6811b7bf-d108-486b-97ce-56150ea971f1)

Drop the following code because it's showing up twice whenever running `crewai`

```
    Options:
    --version: Show the version and exit.   
```